.github/workflows/trivy.yaml

```diff
@@ -0,0 +1,75 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: build
+
+on:
+  push:
+    branches: [ "main", master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "main" ]
+  schedule:
+    - cron: '31 19 * * 1'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    
+    name: Build
+    runs-on: "ubuntu-22.04"
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # - name: Run Trivy vulnerability scanner in repo mode
```

Comment:
This can be removed right? The commented stuff?

Reply:
done